packages/terra-form-input/src/Input.jsx

```diff
@@ -154,6 +148,10 @@ class Input extends React.Component {
       attributes.defaultValue = defaultValue;
     }
 
+    if (attributes.placeholder) {
+      attributes.placeholder = null;
+    }
```

Comment:
This is a more aggressive approach than we usually take. Generally we allow consumers to pass native attributes through the custom props even if they are not officially supported/listed on the documentation site. I think we'll need to document the input does not allow a placeholder (and why) since this is a strict detail. Otherwise we'll want to allow the placeholder to pass through as a custom prop, but not provide any first class API for it/log a warning if it is detected.

Reply:
I have added a `Note` on the doc pages for the component not allowing placeholder.

Reply:
@StephenEsser this is a special case where UX intentionally not only wants to discourage use of placeholders, but disallow it as an accessibility best practice so consumers aren't still reliant on using them, particularly if they don't have direct support from UX and able to get guidance.  Documentation will be the best approach, I think as we continue to improve the docsite, having standout sections similar to Mozilla and W3C that calls out special details or best practice notes will be a good place to call this out.

Reply:
+1 to being heavy handed.  Let's document that this is the UX requirement though and that a passed in placeholder will be dropped.